CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Fixed graphical glitches when resizing on Wayland.
 - On Windows, fix freezes when performing certain actions after a window resize has been triggered. Reintroduces some visual artifacts when resizing.
 - Updated window manager hints under X11 to v1.5 of [Extended Window Manager Hints](https://specifications.freedesktop.org/wm-spec/wm-spec-1.5.html#idm140200472629520).
+- Added `with_gtk_theme_variant` to X11-specific `WindowBuilder` functions
```

Comment:
It's conventional in the CHANGELOG to fully qualify this, i.e. `WindowBuilderExt::with_gtk_theme_variant`. Ending in a period would also be nice.